website/internal/views/__init__.py

```diff
@@ -16,7 +20,28 @@ def triggers():
 def index():
     form = AddNewRoomForm()
     rooms = data_interface.get_user_default_rooms()
-    return render_template("internal/home.html", rooms=rooms, new_room_form=form)
+    devices = []
+    len_rooms = [len(rooms)]
+    room_no = [0]
+    for room in rooms:
+        devices.append(data_interface.get_room_devices(room['room_id']))
+    for room in devices:
+        for device in room:
+            if devices[0][0]['status']['last_temperature'] < 5:
```

Comment:
This doesn't work if there's a non-thermostat in the room. Can you add a check for that?
Also, you probably want one colour per room? So you might want to store colour in the room object and pass these to the template?